static/app/utils/useOverlay.tsx

```diff
@@ -1,16 +1,18 @@
 import {useMemo, useRef, useState} from 'react';
 import {PopperProps, usePopper} from 'react-popper';
 import {detectOverflow, Modifier, preventOverflow} from '@popperjs/core';
-import {useButton} from '@react-aria/button';
+import {useButton as useButtonAria} from '@react-aria/button';
 import {
   AriaOverlayProps,
   OverlayTriggerProps,
-  useOverlay as useAriaOverlay,
-  useOverlayTrigger,
+  useOverlay as useOverlayAria,
+  useOverlayTrigger as useOverlayTriggerAria,
```

Comment:
Adding `…Aria` suffixes to better distinguish react-aria's hooks/return props from PopperJS's.